### PR TITLE
bpo-33061: Add missing 'NoReturn' to __all__ in typing.py

### PR DESCRIPTION
**Pull request by aetracht**
https://bugs.python.org/issue33061

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add missing 'NoReturn' to __all__ in typing.py
```
Comment (ilevkivskyi) [inline Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst]: I think both `NoReturn` and `__all__` should be in double backticks, like this ` ``__all__`` `.

**Review by ilevkivskyi — COMMENTED** [review]
Thank you! Just one small comment.

**Comment by aetracht** [comment]
I'm not sure why my CLA isn't found.  Here's a copy:

[Python Contributor Agreement Form - signed.pdf](https://github.com/python/cpython/files/1825574/Python.Contributor.Agreement.Form.-.signed.pdf)

File: Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Add missing ``'NoReturn'`` to ``__all__`` in typing.py
```
Comment (ilevkivskyi) [inline Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst]: The inner single quotes `'...'` around `NoReturn` are not needed and can be removed.

**Review by ilevkivskyi — COMMENTED** [review]
Have you added your GitHub account name in your account on bugs.python.org?

File: Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst (hunk 2 above)
Comment (aetracht) [reply Misc/NEWS.d/next/Library/2018-03-16-16-07-33.bpo-33061.TRTTek.rst]: Done

**Comment by aetracht** [comment]
Yes, https://bugs.python.org/user28094 shows my GitHub username as `aetracht`.  I'll try changing the email there to my work email (matching my primary GitHub email) and see if that helps.

**Comment by ilevkivskyi** [comment]
It looks like your form was not yet processed, it doesn't appear on your b.p.o. page. I will merge this now, since it is a minor contribution and I have seen your signed form. Could you please also make a similar PR to the [typing repo](https://github.com/python/typing) that contains the backports to older Python 3 versions and to Python 2?

**Pull request merged**